modules/runner/src/main/java/org/apache/ignite/internal/configuration/storage/DistributedConfigurationStorage.java

```diff
@@ -268,6 +270,23 @@ public CompletableFuture<Boolean> write(Map<String, ? extends Serializable> newV
 
         operations.add(Operations.put(MASTER_KEY, ByteUtils.longToBytes(curChangeId)));
 
+        // Condition for a valid MetaStorage data update. Several possibilities here:
```

Comment:
Did you refine the comment according to your changes?

Reply:
Done